app/org/sagebionetworks/bridge/config/BridgeSpringConfig.java

```diff
@@ -199,6 +199,9 @@ public JedisPool jedisPool(BridgeConfig config) {
         // configure Jedis pool
         JedisPoolConfig poolConfig = new JedisPoolConfig();
         poolConfig.setMaxTotal(config.getPropertyAsInt("redis.max.total"));
+        poolConfig.setTestOnBorrow(true);
```

Comment:
Not sure of the value of this. The doc says, "Using the provided factory's PoolableObjectFactory.validateObject(T) method".  But what exactly does it do?  What's the performance overhead?  And if the connection is invalid, does it imply a problem somewhere else?